Tests/RulesTests+GuardSpacing.swift

```diff
@@ -0,0 +1,156 @@
+// Created by @NikeKov on 27.07.2024
+// Copyright © 2024 Nick Lockwood. All rights reserved.
+
+import XCTest
+@testable import SwiftFormat
+
+class GuardSpacingTests: RulesTests {
```

Comment:
The name of the test class should match the rule name

```suggestion
class SpaceWithGuardTests: RulesTests {
```

Reply:
Renamed